canonical-websites/insights.ubuntu.com
# Optimise the size of all pages

## Done
Route images via cloudinary to reduce the size and cache the images.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8023/](http://0.0.0.0:8023/)
- check that the page weight is hugely reduced

## Issue / Card
Fixes #405
